apache/superset
# chore: Removes the ChartIcon component

### SUMMARY
This PR removes the `ChartIcon` component and replaces the code with an icon from our `Icons` component.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<img width="1430" alt="Screen Shot 2021-10-25 at 11 30 51 AM" src="https://user-images.githubusercontent.com/70410625/138715337-f65a00b0-3bf2-420b-8fa1-badfa5522ce7.png">

<img width="1427" alt="Screen Shot 2021-10-25 at 11 27 16 AM" src="https://user-images.githubusercontent.com/70410625/138715370-9b04ff51-64fa-4614-b48e-76091855deff.png">

### TESTING INSTRUCTIONS
1 - Open a dashboard
2 - Enter in edit mode
3 - Click on the 3 dots on the right top corner
4 - Click on Set filter mapping
5 - See the changes  

### ADDITIONAL INFORMATION
